Makefile

```diff
@@ -0,0 +1,2 @@
+testCoverage:
+	docker exec -it djangoplicity-customsearch-web pytest tests --cov=djangoplicity
```

Comment:
Again, please keep homogeneity by running the tests the same way in all the parts, then use here the django tests instead: `./manage.py test`